modules/govuk_jenkins/templates/jobs/mongo_data_sync.yaml.erb

```diff
@@ -0,0 +1,30 @@
+---
+- job:
+    name: mongo_data_sync 
+    display-name: Mongo Data-Sync Job
+    project-type: freestyle
+    description: |
+            This job will perform a mongo data-sync.
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    builders:
+      - shell: |
+          #!/usr/bin/env bash
+          set -x
+
+          # Create an array of mongo cluster members.
+          members=$(govuk_node_list -c mongo)
+
+          # Iterate over the array.
+          # If the member is a master, then iitiate data-sync.
+          for member in ${members[@]};do
```

Comment:
There's a missing space between `;` and `do`.

Reply:
corrected